From ea2eaa9301aa3934188846b99aeaaa168a3f955c Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 17 Jun 2020 11:38:50 +0000
Subject: IWYU: include bitset in VP9ReferenceFrameVector

Bug: 819294
Change-Id: I0b02f301bfc7fc16ba22e29ba0740d9cd83d708d
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2246591
Commit-Queue: Miguel Casas <mcasas@chromium.org>
Reviewed-by: Miguel Casas <mcasas@chromium.org>
Cr-Commit-Position: refs/heads/master@{#779269}
---
 media/gpu/vp9_reference_frame_vector.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/media/gpu/vp9_reference_frame_vector.cc b/media/gpu/vp9_reference_frame_vector.cc
index dc279980ebac..f4541bf53bd2 100644
--- a/media/gpu/vp9_reference_frame_vector.cc
+++ b/media/gpu/vp9_reference_frame_vector.cc
@@ -4,6 +4,8 @@
 
 #include "media/gpu/vp9_reference_frame_vector.h"
 
+#include <bitset>
+
 #include "media/gpu/vp9_picture.h"
 
 namespace media {
-- 
2.20.1

